Zend/Zend.m4

```diff
@@ -422,16 +414,7 @@ AC_RUN_IFELSE([AC_LANG_SOURCE([[
 #include <math.h>
 #include <stdlib.h>
 
-#ifdef HAVE_ISINF
```

Comment:
According to C99 the `INFINITY` macro should always exist, so it should be possibly to adjust the `_zend_get_inf` implementation to always use that and drop the two autoconf checks for infinity representations here.

Reply:
Similarly for `NAN`. That one is not guaranteed to exist, but as PHP assumes it's running on a platform with IEEE 754 compliant floats, it should be defined on platforms we target.

Reply:
(I would suggest doing this separately, before handling zend_isinf and friends, as it removes some of the more ugly users...)